geode-core/src/test/java/org/apache/geode/internal/cache/execute/PRSetOperationJTADUnitTest.java

```diff
@@ -169,6 +179,7 @@ private void verifyRegionKeysetWithJTA(boolean disableSetOpToStartJTA, boolean i
       userTX.begin();
       Collection<Long> set = region.keySet();
       set.forEach((key) -> assertTrue(testData.keySet().contains(key)));
+      testData.keySet().forEach((key) -> assertTrue(set.contains(key)));
```

Comment:
Better would have been a size check and then checking the key from two sets.

Reply:
The size check will start the transaction. When set operation does not start JTA, we can not check the size. There are size checks in other cases.